Packs/KnowBe4KMSAT/Integrations/KnowBe4KMSAT/KnowBe4KMSAT.py

```diff
@@ -0,0 +1,1348 @@
+
+    Returns:
+        CommandResults: Returns context data for training campaigns
+    """
+    params = get_pagination(args)
+    response = client.kmsat_training_campaigns(params)
+    markdown = tableToMarkdown(
+        "Training Campaigns",
+        response,
+        [
+            "campaign_id",
+            "name",
+            "groups",
+            "status",
+            "content",
+            "duration_type",
+            "start_date",
+            "end_date",
+            "relative_duration",
+            "auto_enroll",
+            "allow_multiple_enrollments",
+            "completion_percentage",
+        ],
+    )
+
+    return CommandResults(
+        outputs_prefix="KMSAT.TrainingCampaigns",
+        outputs_key_field="campaign_id",
+        raw_response=response,
+        outputs=response,
+        readable_output=markdown,
+    )
+
+
+def kmsat_training_enrollments_list_command(
+    client: Client, args: dict
+) -> CommandResults:
+    """ Lists KMSAT training enrollments
+
+    Args:
+        client (Client): Report Client
+        args (dict): Params for training enrollments
+
+    Raises:
+        DemistoException: Raises Demisto Exception
+
+    Returns:
+        CommandResults: Returns context data for training enrollments
+    """
+    status = remove_empty_elements(args.get("status"))
+    params = get_pagination(args)
+    response = client.kmsat_training_enrollments(params)
+
+    data = []
+    filtered_items_in_page = 0
+    items_total = len(response)
+
+    # Sets paging_end False if the response count is less than the per_page
+    per_page = int(params.get('per_page')) if (params.get('per_page')) else 100
+
+    paging_end = len(response) < per_page
+
+    # Adds only the filtered items to the response with counts
+    if status is not None:
+        for i in range(len(response)):
+            if response[i]['status'] == f"{status}":
+                data.append(response[i])
+                filtered_items_in_page += 1
+    else:
+        data = client.kmsat_training_enrollments(params)
+
+    # Adds meta to the result set for paging
+    metadata = {
+        "paging_end": paging_end,
+        "filtered_items_in_page": filtered_items_in_page,
+        "items_total": items_total
+    }
+
+    d = {
+        "data": data,
+        "meta": metadata
+    }
+
+    markdown = tableToMarkdown(
+        "Training Enrollments",
+        d["data"],
+        [
+            "enrollment_id",
+            "content_type",
+            "module_name",
+            "user",
+            "campaign_name",
+            "enrollment_date",
+            "start_date",
+            "completion_date",
+            "status",
+            "time_spent",
+            "policy_acknowledged",
+        ]
+    )
+
+    if not data:
+        raise DemistoException(
+            "Translation failed: the response from server did not include `kmsat_training_enrollments_list_command`.",
+            res=data,
+        )
```

Comment:
What about this exception? what does it mean that the server did not include the command?

Reply:
Good catch.  Sorry, I missed this one.